qiskit/circuit/library/overlap.py

```diff
@@ -62,7 +62,7 @@ def __init__(
     ):
         """
         Args:
-            unitary1: Unitary acting on the ket vector.
+            unitary1: Unitary acting on the key vector.
```

Comment:
"ket" is actually the correct word here. It's a term used in quantum mechanics. You can find the details on it here: https://en.wikipedia.org/wiki/Bra%E2%80%93ket_notation